iOS-Email-Client/Shared/NewEmailHandler.swift

```diff
@@ -256,6 +256,7 @@ class NewEmailHandler {
                 if(fromContact?.spamScore ?? 0 >= 2){
                     let spamLabel = SharedDB.getLabel(SystemLabel.spam.id)
                     email.labels.append(spamLabel!)
+                    APIManager.postReportContact(emails: [fromContact!.email], type: ContactUtils.ReportType.spam, data: nil, token: myAccount.jwt, completion: {_ in })
```

Comment:
Are you checking if the incoming `Labels` of the email do not contain `Spam`? We need to check that it does, becuase if it does, we don't need to report the contact